data/json/items/generic/casing.json

```
@@ -25,17 +25,17 @@
     "copy-from": "casing",
     "looks_like": "9mm_casing",
     "type": "GENERIC",
-    "name": { "str": ".22 LR casing" },
-    "description": "An empty casing from a .22 LR round.  These can't be hand-reloaded.",
+    "name": { "str": ".22 casing" },
+    "description": "An used, empty casing from a .22 round.",
```

Also smol grammar fix.
```suggestion
    "description": "A used, empty casing from a .22 round.",
```